pkgs/os-specific/linux/bluez/bluez5_28.nix

```diff
@@ -51,6 +51,7 @@ in stdenv.mkDerivation rec {
   # FIXME: Move these into a separate package to prevent Bluez from
   # depending on Python etc.
   postInstall = ''
+    cp ./attrib/gatttool $out/bin/gatttool
```

Comment:
the version of bluez you are currently modifying is only used by libreoffice. Is this indented? Otherwise there is bluez5.nix in the same directory